nucliadb/src/nucliadb/search/search/query_parser/parser.py

```diff
@@ -74,7 +74,7 @@ def _parse_rank_fusion(self) -> RankFusion:
         rank_fusion: RankFusion
 
         top_k = self._parse_top_k()
-        window = top_k
+        window = min(top_k, 500)
```

Comment:
A new hard limit? Why doesn't it match the api limit?

Reply:
It is the same on the API

Reply:
500 rank fusion, 200 reranker